test/pkgcheck.sh

```diff
@@ -1,10 +1,72 @@
 #!/bin/sh
-# Verify that the various build systems produce identical results on a Unixlike system
+
+usage() {
+  cat <<"_EOF_"
+Usage: sh test/pkgcheck.sh [--zlib-compat]
+
+Verifies that the various build systems produce identical results on a Unixlike system.
+If --zlib-compat, tests with zlib compatible builds.
+
+To cross-build, install the appropriate qemu and gcc packages,
+and set the environment variables used by configure or cmake, e.g.
+
+armel:
+$ sudo apt install ninja-build diffoscope qemu gcc-arm-linux-gnueabihf libc-dev-armel-cross
+$ export CHOST=arm-linux-gnueabihf
+$ export CMAKE_ARGS="-DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-arm.cmake -DCMAKE_C_COMPILER_TARGET=${CHOST}"
+
+aarch64:
+$ sudo apt install ninja-build diffoscope qemu gcc-aarch64-linux-gnu libc-dev-arm64-cross
+$ export CHOST=aarch64-linux-gnu
+$ export CMAKE_ARGS="-DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-aarch64.cmake -DCMAKE_C_COMPILER_TARGET=${CHOST}"
+
+ppc (32 bit big endian):
+$ sudo apt install ninja-build diffoscope qemu gcc-powerpc-linux-gnu libc-dev-powerpc-cross
+$ export CHOST=powerpc-linux-gnu
+$ export CMAKE_ARGS="-DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-powerpc.cmake"
+
+ppc64le:
+$ sudo apt install ninja-build diffoscope qemu gcc-powerpc64le-linux-gnu libc6-dev-ppc64-powerpc-cross
```

Comment:
Should be `libc6-dev-ppc64el-cross`. And also replace `libc` by `libc6` on the other package names.